src/main/scala/org/ergoplatform/modifiers/mempool/ErgoTransaction.scala

```diff
@@ -96,27 +96,35 @@ case class ErgoTransaction(override val inputs: IndexedSeq[Input],
   /**
     * @return total computation cost
     */
-  def statefulValidity(boxesToSpend: IndexedSeq[ErgoBox], blockchainState: ErgoStateContext): Try[Long] = {
-    lazy val lastUtxoDigest = AvlTreeData(blockchainState.digest, ErgoBox.BoxId.size)
+  def statefulValidity(boxesToSpend: IndexedSeq[ErgoBox],
+                       blockchainState: ErgoStateContext,
+                       metadata: Metadata): Try[Long] = {
```

Comment:
I thought we decided to exclude metadata from validation?

Reply:
There is an issue for that (https://github.com/ScorexFoundation/sigmastate-interpreter/issues/267), but dependency is still in the sigma code we use